index.html

```diff
@@ -20,97 +20,97 @@ <h3>Lopper</h3>
         </ul>
     </header>
     <main>
-        <div>
-            <h1>About Us</h1>
-            <p>WE BUILD GREAT BUSINESS</p>
-        </div>
-        <section>
-            <p>WHAT WE DO</p>
-            <h2>Apparently we had reached a great height in the 
+        <section class="main_title">
+            <h1 class="main_text">About Us</h1>
+            <p class="main_paragraph">WE BUILD GREAT BUSINESS</p>
+        </section>
+        <section class="main_about">
+            <p class="about_subtitle">WHAT WE DO</p>
+            <h2 class="about_text">Apparently we had reached a great height in the 
                 atmosphere, for the sky was a dead black, and the stars 
                 had ceased to twinkle.</h2>
-            <div>
+            <div class="sub_about">
                 <div>
-                    <h4>Our objectives</h4>
-                    <p>Amet consectetur adipiscing elit pellentesque habitant morbi. Fermentum et sollicitudin ac orci phasellus. Dolor sit amet consectetur adipiscing elit duis. Euismod quis viverra nibh cras pulvinar mattis nunc sed blandit.</p>
+                    <h4 class="sub_subtitle">Our objectives</h4>
+                    <p class="about_subtext">Amet consectetur adipiscing elit pellentesque habitant morbi. Fermentum et sollicitudin ac orci phasellus. Dolor sit amet consectetur adipiscing elit duis. Euismod quis viverra nibh cras pulvinar mattis nunc sed blandit.</p>
                 </div>
                 <div>
-                    <h4>Our Story</h4>
-                    <p>Etiam dignissim diam quis enim lobortis. Egestas sed sed risus pretium quam vulputate dignissim. Eleifend quam adipiscing vitae proin sagittis. Pharetra pharetra massa massa ultricies. Elementum eu facilisis sed odio morbi. Morbi tincidunt augue interdum velit.</p>
+                    <h4 class="sub_subtitle">Our Story</h4>
+                    <p class="about_subtext">Etiam dignissim diam quis enim lobortis. Egestas sed sed risus pretium quam vulputate dignissim. Eleifend quam adipiscing vitae proin sagittis. Pharetra pharetra massa massa ultricies. Elementum eu facilisis sed odio morbi. Morbi tincidunt augue interdum velit.</p>
                 </div>
             </div>
         </section>
-        <section>
-           <p>WHAT WE DO</p>
-           <h2>What we do</h2> 
+        <section class="">
+           <p class="">WHAT WE DO</p>
```

Comment:
Não é indicado escrever frases em maiúsculo direto no HTML, leitores de tela irão ler cada letra individualmente. O melhor nessa caso é deixar todo o texto normal (What We Do/What we do) e inserir nessa tag uma classe "uppercase", no CSS para transformar um texto em maiúsculo é só adicionar a propriedade `text-transform` com o valor `uppercase`